aioca/_catools.py

```diff
@@ -758,8 +761,9 @@ async def caput(pv: str, value, datatype=None, wait=False):
     return CANothing(pv)
 
 
-@caput.register  # type: ignore
-async def caput_array(pvs: list, values, repeat_value=False, **kwargs):
+@caput.register(list)  # type: ignore
+@caput.register(tuple)  # type: ignore
+async def caput_array(pvs: List[str], values, repeat_value=False, **kwargs):
```

Comment:
```suggestion
async def caput_array(pvs: PVS, values, repeat_value=False, **kwargs):
```

Reply:
Not that it really matters, as the overload is what mypy looks at, but we might as well be correct about these things...